SQL/Archive/19.1/2017-04-24-UseOurIssueTrackerPl0x.sql

```diff
@@ -0,0 +1,64 @@
+INSERT INTO `ConfigSettings`
```

Comment:
@driusan 
FYI this patch does not seem to be in any 19.1/ directory (I checked major and 19.1-release branches)...